client/client.go

```diff
@@ -72,6 +75,23 @@ type WrapperV2 struct {
 
 	// commandLine is the command line use to execute gsctl, can be overridden.
 	commandLine string
+
+	// rawClient is a client used to make simple, authenticated HTTP requests
+	// against the Giant Swarm API using Go's net/http API.
+	rawClient *http.Client
+}
+
+// ClusterStatus is a type we use to unmarshal a cluster status JSON response
+// from the API. Note: this is scarce, leaving out many available details.
+type ClusterStatus struct {
+	Cluster *ClusterStatusCluster `json:"cluster,omitempty"`
+}
+type ClusterStatusCluster struct {
+	Nodes []*ClusterStatusClusterNode `json:"nodes,omitempty"`
+}
+type ClusterStatusClusterNode struct {
```

Comment:
Why don't we import `apiextensions` here and use these types? This is what defines them.

Reply:
Done